### PR TITLE
Update to jasmine-2.3.4

### PR DESCRIPTION
**Pull request by andyholmes**
This is the latest patch release from the 2.3.4 series

### CODE REVIEW

**Comment by andyholmes** [comment]
This was my first go (before bed :wink:). Below is the reject patch snippets, I'm not sure why those were there so advice would be welcome.

`meson test` failed, but it seems to be because of errant output on stdout/stderr because all tests actually pass with `ok`.

```diff
--- jasmine.js	2017-02-07 17:19:05.530588235 -0800
+++ jasmine.js	2017-02-07 17:06:36.825849025 -0800
@@ -689,12 +696,6 @@
 
       runnableLookupTable[suite.id] = suite;
       return suite;
-
-      function suiteStarted(suite) {
-        currentlyExecutingSuites.push(suite);
-        defaultResourcesForRunnable(suite.id, suite.parentSuite.id);
-        reporter.suiteStarted(suite.result);
-      }
     };
 
     this.describe = function(description, specDefinitions) {
@@ -1106,8 +1107,6 @@
       }
     };
 
-    return self;
-
     function legacyIE() {
       //if these methods are polyfilled, apply will be present
       return !(realTimingFunctions.setTimeout || realTimingFunctions.setInterval).apply;
@@ -3055,5 +3058,5 @@
 };
 
 getJasmineRequireObj().version = function() {
-  return '2.2.0';
+  return '2.2.1';
 };
```

**Comment by ptomato** [comment]
I've made the 2.3.0 release and updated the [release checklist](https://github.com/ptomato/jasmine-gjs/blob/master/release-checklist.md) so it's actually correct now, so have at it!

File: jasmine-gjs.spec [hunk 1]
```diff
@@ -46,6 +46,8 @@ be displayed in your terminal.
 
 
 %changelog
+* Mon Aug 26 2020 Andy Holmes <andrew.g.r.holmes@gmail.com> - 2.3.4-1
```
Comment (ptomato) [inline jasmine-gjs.spec:49]: It's Wednesday, but no worries, I'll fix it up.

**Review by ptomato — APPROVED** [review]
Thanks!

File: jasmine-gjs.spec (hunk 1 above)
Comment (andyholmes) [reply jasmine-gjs.spec:49]: oops :D

**Pull request merged**